src/patternfly/components/DataList/data-list-icon.hbs

```diff
@@ -1,3 +1,6 @@
-<i class="pf-c-data-list__icon fas{{#if data-list-icon--type}} fa-{{data-list-icon--type}}{{/if}}"
-aria-hidden="true">
-</i>
+<span class="pf-c-data-list__icon{{#if data-list-icon--modifier}} {{data-list-icon--modifier}}{{/if}}"
```

Comment:
do you know why we use this element? we don't style it and it's already in an element `.pf-c-data-list__cell.pf-m-icon`

Reply:
Yep good point we can safely remove